src/Behat/Testwork/Autoloader/ServiceContainer/AutoloaderExtension.php

```diff
@@ -78,7 +78,9 @@ public function configure(ArrayNodeDefinition $builder)
             ->treatTrueLike($this->defaultPaths)
             ->treatNullLike(array())
             ->treatFalseLike(array())
-
+            ;
```

Comment:
The return value of `treatFalseLike()` is a NodeDefinition, which does not have the `prototype()` function, so we need to narrow down the actual type